docs/Presentaciones/02-04-2024.md

```diff
@@ -0,0 +1,242 @@
+
+
+<p align="center">
+<img src="\img\pptx\relojglob.png" alt="relojglob" width="60%" height="60%"/>
+Figura 35. Reloj del proyecto global
+</p>
+
+Y este es el reloj global que está 300 horas por encima del esperado, debido a la migración de los documentos a docusaurus y al cambio a mongo.
+
+<p align="center">
+<img src="\img\pptx\alberto.png" alt="alberto" width="60%" height="60%"/>
+Figura 36. Alberto.
+</p>
+
+**Además, nos gustaría felicitar a alberto por haber resuelto durante este sprint tantos problemas de código de los que dependia la gente de frontend. Muchas gracias Alberto**
+
+## Usuarios piloto
+
+<p align="center">
+<img src="\img\pptx\pilot.png" alt="pilot" width="60%" height="60%"/>
+Figura 37. Usuarios piloto
+</p>
+
+**Bueno, continuamos con los usuarios piloto y nuestra interacción este segundo sprint.** Nuestros usuarios piloto son el G1 de mañana y las ONGs, destacando que además de estar contactados, informados e involucrados ya han firmado el CA.
+
+<p align="center">
+<img src="\img\pptx\feedback.png" alt="feedback" width="60%" height="60%"/>
+Figura 38. Aspectos de mejora recogidos
+</p>
+
+**En este sprint hemos recogido el feedbackd de nuestros usuarios piloto, lo hemos analizado y recogido en el registro de cambios, si escaneais el QR podréis acceder a ese apartado del docusaurus para consultarlo.** La mayoría de cambios relacionados con algunos requisitos que no estaban del todo claros y con correcciones en la interfaz de usuario.
+
+<p align="center">
+<img src="\img\pptx\calendar1.png" alt="calendar1" width="60%" height="60%"/>
+Figura 39. Calendario de Marzo
+</p>
+
+Este es el calendario de Marzo con la ventana en la que entregamos a los usuarios piloto nuestro servicio y el día de análisis de las solicitudes.
+
+<p align="center">
+<img src="\img\pptx\calendar2.png" alt="calendar2" width="60%" height="60%"/>
+Figura 40. Calendario de Abril
+</p>
+
+Este es el calendario de abril (le hemos dado esta semana la app a los usuarios piloto).
+
+<p align="center">
+<img src="\img\pptx\calendar3.png" alt="calendar3" width="60%" height="60%"/>
+Figura 41. Calendario de Mayo
+</p>
+
+Y este es el calendario de mayo.
+
+<p align="center">
+<img src="\img\pptx\infoG1.png" alt="infoG1" width="60%" height="60%"/>
+Figura 42. Aspectos clave del CA G1
+</p>
+
+**Estos son algunos aspectos clave de nuestro CA con el grupo 1**: cumplir las deadlines y tener una comunicación fluida son comunes a los dos CA, y  se diferencian del de las ONGs en que deben usar iTop, clockify y que en caso de que no nos guste el tipo de feedback que nos dan, se comprometen a cambiar de revisor.
+
+<p align="center">
+<img src="\img\pptx\infoONG.png" alt="infoONG" width="60%" height="60%"/>
+Figura 43. Aspectos clave del CA ONGs
+</p>
+
+**Y aquí los de la ONG que sólo deben probar el prototipo correspondiente a su ONG**
```

Comment:
Can you phrase this in a more natural way, mostly the start of the sentences, they feel too stiff